### ISSUE TITLE
Evaluate Build Online Courses & E-Learning Platforms with Next.js

### ISSUE THREAD

**Issue by lynnchsu** [lynncco/oss#8]
https://www.youtube.com/playlist?list=PLB6KpVV9FHlJ0QHOAJriehk-jWGZsF4bF
https://github.com/ebouJ/E-learning

**Comment by lynnchsu** [comment]
Tutorial by another YouTuber for building library with Shadcn
Let's Build a Component Library Using Shadcn's UI Components
https://www.youtube.com/watch?v=ChjoKSNWiBo
https://github.com/rajeshdavidbabu/shadcn-ui-lib-starter

Only 3 simple component but he shows the ins and outs of using Shadcn as base of component library


![Image](https://github.com/lynncco/oss/assets/140572240/954ba17a-25f1-42d2-bacc-ca1df0e98d64)

**Comment by hiraaziz** [comment]
Yes I looked at this oss project, It has all the basic features if we go with this oss, we might need few new features.
